src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -121,5 +121,10 @@
             <argument type="service" id="twig" on-invalid="ignore" />
             <argument type="service" id="templating" on-invalid="ignore" />
         </service>
+
+        <service id="routing.route_loader_container" class="Symfony\Bundle\FrameworkBundle\Routing\RouteLoaderContainer">
```

Comment:
shouldn't we replace this by an inline service, so that we'll guarantee nobody can reference the service?

Reply:
You mean this : https://symfony.com/doc/current/service_container/alias_private.html#anonymous-services right? I didn't even know it existed 😁 Looks like a very good idea to me!